### PR TITLE
CIV-0000 DTSCCI-208: Multiple Incidents: Defendant Can't link claim

### PR DESCRIPTION
**Pull request by srochani**
**Before creating a pull request make sure that:**

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)

Please remove this line and everything above and fill the following sections:

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

File: src/main/java/uk/gov/hmcts/reform/civil/service/claimstore/ClaimStoreService.java [hunk 1]
```diff
@@ -42,6 +43,15 @@ public List<DashboardClaimInfo> getClaimsForDefendant(String authorisation, Stri
         }
     }
 
+    public DefendantLinkStatus isOcmcDefendantLinked(String caseReference) {
+        try {
+            return claimStoreApi.isDefendantLinked(caseReference);
+        } catch (Exception e) {
+            log.error(e.getMessage());
```
Comment (linusnorton) [inline src/main/java/uk/gov/hmcts/reform/civil/service/claimstore/ClaimStoreService.java:50]: maybe pass e to log.error to keep the original exception

File: src/main/java/uk/gov/hmcts/reform/civil/service/pininpost/DefendantPinToPostLRspecService.java [hunk 2]
```diff
@@ -91,4 +94,11 @@ public String validateOcmcPin(String pin, String caseReference) {
             throw new PinNotMatchException();
         }
     }
+
+    public boolean isOcmcDefendantLinked(String caseReference) {
+        log.info("-----------calling ocmc isDefendantLinked()-------------");
```
Comment (linusnorton) [inline src/main/java/uk/gov/hmcts/reform/civil/service/pininpost/DefendantPinToPostLRspecService.java:99]: is this really necessary?

**Review by rishikrsharma — APPROVED** [review]

File: src/main/java/uk/gov/hmcts/reform/civil/service/pininpost/DefendantPinToPostLRspecService.java (hunk 2 above)
Comment (srochani) [reply src/main/java/uk/gov/hmcts/reform/civil/service/pininpost/DefendantPinToPostLRspecService.java:99]: Not required, removing the log

File: src/main/java/uk/gov/hmcts/reform/civil/service/claimstore/ClaimStoreService.java (hunk 1 above)
Comment (srochani) [reply src/main/java/uk/gov/hmcts/reform/civil/service/claimstore/ClaimStoreService.java:50]: Ah, will add e

**Review by linusnorton — APPROVED** [review]

**Review by nigeldunne — DISMISSED** [review]

**Review by neeta-hmcts — APPROVED** [review]

**Review by sankhajuria — APPROVED** [review]

**Comment by srochani** [comment]
closing and reopening the PR

**Pull request merged**